src/tools/annotation/bidirectionalTool/moveHandle/getLineVector.js

```diff
@@ -0,0 +1,12 @@
+export default function getLineVector(cps, rps, pointA, pointB) {
```

Comment:
Please use more descriptive names. JSDoc also appreciated.

A little surprised we don't have _something_ like this already lying around.

Reply:
I renamed the parameters of that function, moved it under `utils` folder and added JSDoc.
Actually we have a lot of bugs related to column and row pixel spacing in other tools.
Using the image with non-square pixels we are able to see a lot of bugs in other tools (even on simple tools like angle).